activerecord/lib/active_record/railties/job_runtime.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "active_record/log_subscriber"
+
+module ActiveRecord
+  module Railties # :nodoc:
+    module JobRuntime #:nodoc:
+      private
+        def instrument(operation, payload = {}, &block)
+          if operation == :perform && block
```

Comment:
I thought about omitting `operation == :perform` here to capture `db_runtime` whenever there is a block which could be touching the database.  That thought also makes me wonder if there is a more general pattern that's trying to emerge here.